features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx

```diff
@@ -347,6 +347,8 @@ class ApiConsole extends React.Component {
             return 'API Not found !';
         }
 
+        const {authorizationHeader='Authorization'} = api;
```

Comment:
a space is required after { and before }

Reply:
It has already been overwritten by another fix.